FluidNC/src/Maslow/Maslow.cpp

```diff
@@ -574,8 +594,8 @@ void Maslow_::safety_control() {
   MotorUnit* axis[4] = { &axisTL, &axisTR, &axisBL, &axisBR };
   for (int i = 0; i < 4; i++) {
       //If the current exceeds some absolute value, we need to call panic() and stop the machine
-      if (axis[i]->getMotorCurrent() > currentThreshold+2200  && !tick[i]) {
-          log_error("Motor current on " << axis_id_to_label(i).c_str() << " axis exceeded threshold of " << currentThreshold+2200
+      if (axis[i]->getMotorCurrent() > currentThreshold+220000  && !tick[i]) {
```

Comment:
I was running into issues with this tripping so I guess I just disabled it here. Not sure what we should do about that